app/templates/components/event-card.hbs

```diff
@@ -20,11 +20,11 @@
       {{/smart-overflow}}
       <div class="meta">
         <span class="date">
-          {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
+          {{moment-format event.startsAt 'ddd, MMM DD HH:mm A'}}
         </span>
       </div>
       {{#smart-overflow class='description'}}
-        {{event.shortLocationName}}
```

Comment:
@niranjan94 Had confused it with the class thought it should be here 😅 apologies, I have changed it back.